koku/api/report/queries.py

```diff
@@ -106,6 +108,25 @@ def sanitize_tag(tag):
     return sanitized_tag
 
 
+def restore_tag_alias(value, alias_lookup, prefix="No-"):
+    """
+    Restore the original tag value from an alias.
+    Example:
+        "Tirea_app_group" → "Tirea/app-group"
+        "No-Tirea_app_group" → "No-Tirea/app-group"
+    """
+    if not isinstance(value, str):
```

Comment:
Can you give me more context about what we are trying to accomplish with this function